node/packages/parser/src/ethereum/__tests__/mockData/ethOnly.ts

```diff
@@ -1,82 +1,54 @@
-        input:
-          '0x7ff36ab500000000000000000000000000000000000000000016df982add888d233e7e8c000000000000000000000000000000000000000000000000000000000000008000000000000000000000000070336a5a868cf9e0a9d1b47fbe1f521ad6c5c3640000000000000000000000000000000000000000000000000000000060cd0b8f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc200000000000000000000000095ad61b0a150d79219dcf64e1e6cc01f0b64c4ce',
-        hash: '0xc3acd7001e1d01fe4b451e33e24ce399ee533a18d00b441e3bb6290209b072e1',
-        blockNumber: '',
-        from: '0x70336A5A868cF9E0a9D1B47fbE1F521aD6C5C364',
-        transactionIndex: '',
-      },
+import { mempoolMock } from './mempoolMock'
+
+const ethOnly = {
+  txid: '0x5557426ee8e2f3da4703a4b5ab265e5d3a5626da1f69b4d048c7230a75fa5936',
+  vin: [
+    {
+      n: 0,
+      addresses: ['0x8C8D7C46219D9205f056f28fee5950aD564d7465'],
+      isAddress: true,
     },
-    ethereumSpecific: {
-      status: -1,
-      nonce: 6,
-      gasLimit: 140096,
-      gasUsed: null,
-      gasPrice: '24200000000',
-      data:
-        '0x7ff36ab500000000000000000000000000000000000000000016df982add888d233e7e8c000000000000000000000000000000000000000000000000000000000000008000000000000000000000000070336a5a868cf9e0a9d1b47fbe1f521ad6c5c3640000000000000000000000000000000000000000000000000000000060cd0b8f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc200000000000000000000000095ad61b0a150d79219dcf64e1e6cc01f0b64c4ce',
+  ],
+  vout: [
+    {
+      value: '46224000000000000',
+      n: 0,
+      addresses: ['0x05EFB6161fe280dC17E5f9EaB6200b889E2D0A88'],
+      isAddress: true,
     },
+  ],
+  blockHash: '0x39fbd8a96a5b519ea5301ecde6dbf2c63c4a767a3368586ca4c5165ef0999af3',
+  blockHeight: 12659161,
+  confirmations: 14,
+  blockTime: 1624029849,
+  value: '46224000000000000',
+  fees: '1050000000000000',
+  ethereumSpecific: {
+    status: 1,
+    nonce: 4605,
+    gasLimit: 21000,
+    gasUsed: 21000,
+    gasPrice: '50000000000',
+    data: '0x',
   },
-  tx: {
-    txid: '0x5557426ee8e2f3da4703a4b5ab265e5d3a5626da1f69b4d048c7230a75fa5936',
-    vin: [
-      {
-        n: 0,
-        addresses: ['0x8C8D7C46219D9205f056f28fee5950aD564d7465'],
-        isAddress: true,
-      },
-    ],
-    vout: [
-      {
-        value: '46224000000000000',
-        n: 0,
-        addresses: ['0x05EFB6161fe280dC17E5f9EaB6200b889E2D0A88'],
-        isAddress: true,
-      },
-    ],
-    blockHash: '0x39fbd8a96a5b519ea5301ecde6dbf2c63c4a767a3368586ca4c5165ef0999af3',
-    blockHeight: 12659161,
-    confirmations: 14,
-    blockTime: 1624029849,
-    value: '46224000000000000',
-    fees: '1050000000000000',
-    ethereumSpecific: {
-      status: 1,
-      nonce: 4605,
-      gasLimit: 21000,
-      gasUsed: 21000,
-      gasPrice: '50000000000',
-      data: '0x',
+  coinSpecificData: {
```

Comment:
To be clear, `coinSpecificData` only exists on the mempool transaction from blockbook. We do not use this right now, so I would be ok with just removing it if we want to make things easy, but I wouldn't want it looking like it is part of the confirmed tx payload

Reply:
Got it, I was not aware of that! I'll remove it today.

Reply:
Yeah no worries at all. Lot's of random data coming back in these payloads that is not obvious haha

Reply:
Actioned in https://github.com/shapeshift/unchained/pull/283/commits/eb15ccb4880aaf1be529a5cc883f9b2151f44d97